macros/src/lib.rs

```diff
@@ -145,7 +145,7 @@ pub fn await_or_block(expr: TokenStream) -> TokenStream {
         {
             #[cfg(all(not(target_arch = "wasm32"), not(feature = "async-interface")))]
             {
-                tokio::runtime::Runtime::new().unwrap().block_on(#expr)
+                tokio::runtime::Builder::new_current_thread().build().unwrap().block_on(#expr)
```

Comment:
[`Runtime::new()`](https://docs.rs/tokio/0.3.3/tokio/runtime/struct.Runtime.html#method.new) doesn't exist in the `rt` feature, only the `rt-multi-thread` feature.

Therefore, we need we need to [build one manually](https://docs.rs/tokio/0.3.3/tokio/runtime/index.html#current-thread-scheduler).

Reply:
This is probably more efficient since the old one was probably spinning up a multithreaded runtime to wait on a single future.

Reply:
In theory it shouldn't have, because only the `rt-core` feature was enabled and not the `rt-threaded` one. But yeah, it makes more sense to use the current thread itself to block on the future instead of spawning a new one, even if it's a single one.